cfe_internal/enterprise/federation/federation.cf

```diff
@@ -616,9 +642,9 @@ bundle agent imported_data
 # @brief Run script to import dumps on superhub
 {
   commands:
-      "/bin/bash"
+      "/bin/bash" -> { "CFE-951" }
         arglist => {"$(cfengine_enterprise_federation:config.bin_dir)/import.sh"},
-        contain => default:in_shell;
+        contain => default:in_shell_dir( "/tmp" );
```

Comment:
@craigcomstock in what directory is import.sh typically run from?

In the script I see that it's outputting information to things like schema_setup.log but I don't see any changes to the working directory, maybe I am just missing them.

Reply:
Right, this is something I was fixing yesterday in fact. It is just written to cwd which I can confirm on a superhub I have handy. But the scripts really should write to a specific location, which is what I was fixing yesterday. Changing to `/tmp` is fine.

Reply:
Yeah, i was actually asking about this in relation to something completely different (not this PR).

Reply:
I will likely submit a PR "fixing" this "unexplicit" location of these log files.